src/components/form-attachment/list.vue

```diff
@@ -160,6 +174,11 @@ export default {
     },
     dsHashset() {
       return this.datasets.dataExists ? new Set(this.datasets.map(d => `${d.name}.csv`)) : null;
+    },
+    datasetLinkable() {
+      return this.attachments.dataExists &&
+        this.datasets.dataExists &&
+        any(d => this.attachments.data.has(`${d.name}.csv`), this.datasets.data);
```

Comment:
Could we do this in a way that's case-insensitive with respect to `.csv`? Maybe the easiest thing would just be to check for both `${d.name}.csv` and `${d.name}.CSV`.

Also, no need to change anything, but if you used the `some()` method instead of `any()`, you wouldn't need to access the `data` property:

```js
this.datasets.some(d => this.attachments.has(`${d.name}.csv`))
```